box/boxnavigator.py

```diff
@@ -43,12 +43,15 @@ def __init__(self, position: Pt, rotation: float, env: BoxEnv) -> None:
 
         # TODO: find appropriate values for these
         self.distance_threshold = 75
-        self.translation_increment = 120
+        self.forward_increment = None
```

Comment:
Should we add `forward_increment` as an argument for the constructor? Then remove the method?

Reply:
I also made the rotation_increment into an optional argument if one wishes to change them.